src/components/Reviews.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react"
+import { getReviews } from '../api.js'
+import ReviewCard from '../components/ReviewCard'
+
+const Reviews = () => {
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        getReviews()
+        .then((reviews) => {
+            setReviews(reviews)
+        })
+    })
+
+    return (
+        <section>
+            <h2 className="reviewsHeading">Reviews</h2>
+            <ul className="reviewList">
+             {reviews.map((review) => {
+               return <ReviewCard key={review.review_id} review={review}/>
```

Comment:
Really like that you've move the review card out to a seperate component, keeps the code super clean!